CMakeLists.txt

```diff
@@ -72,6 +73,10 @@ if(WITH_LZO)
   find_package(LZO)
 endif()
 
+if(WITH_XCB)
+  find_package(X11) # Need CMake 3.24.0 to find X11 libraries. see https://cmake.org/cmake/help/v3.24/module/FindX11.html 
```

Comment:
Almost! I think it should say XCB instead of X11.

Reply:
changed